docs/changelog.rst

```diff
@@ -1,2 +1,8 @@
 Changelog
 ~~~~~~~~~
+
+0.1.0 (October 2024)
+-------------------
+
+* ``QuESTBackend`` that uses ``pyQuEST`` for running simulations in QuEST.
```

Comment:
Totally optional suggestion below


You should be able to put a 

```
.. currentmodule:: pytket.extensions.quest
```

Directive at tthe top of the page

If you then do `:py:class:QuESTBackend` the crosslink to the docs should be generated.